dialects/postgres.yaml

```diff
@@ -67,8 +67,6 @@ scalar_functions:
   - name: modulus
     local_name: "%"
     infix: True
-    required_options:
```

Comment:
postgres supports modulus with numeric operands, but as per Substrait documentation, it should only work on integer values.

Reply:
We'll need to update the substrait docs here too